app/src/main/java/org/oppia/android/app/topic/revisioncard/RevisionCardFragmentPresenter.kt

```diff
@@ -79,6 +86,60 @@ class RevisionCardFragmentPresenter @Inject constructor(
     return binding.root
   }
 
+  private fun setUpRevisionNavigationCards(
+    binding: RevisionCardFragmentBinding,
+    topicId: String,
+    subtopicId: Int,
+    subtopicListSize: Int
+  ) {
+
+    binding.prevSubtopicImageView.setEntityType(entityType)
+    binding.nextSubtopicImageView.setEntityType(entityType)
+
+    if (subtopicId == 1) {
+      binding.previousNavigationCard.visibility = View.INVISIBLE
+    } else {
+      getReviewCardViewModel().previousSubtopicLiveData.observe(fragment) { previousSubtopic ->
```

Comment:
Why can't these changes be done via databinding? I'd imagine the image view would be set something like:

```xml
app:lessonThumbnail="@{viewModel.previousSubtopic.subtopic.subtopicThumbnail}"
```

And the text:

```xml
app:text="@{viewModel.computeSubtopicTitleText()}"
```

Where ``computeSubtopicTitleText`` does the same translation extra string as below. We do this in other view models as well.

This allows you to manage everything through databinding, and may somewhat minimize code duplication between the previous/next subtopic cards (depending on how you implement it).